versions.tf

```diff
@@ -5,3 +5,7 @@ terraform {
     aws = ">= 3.0.0, < 4.0.0"
   }
 }
+
+provider "aws" {
```

Comment:
Remove this region @Z11mm

Reply:
Just the region or the entire provider block? @MeNsaaH 

I added it to fix a terraform validate error...

Reply:
The entire provider block.

The `AWS_REGION` variable is set on the CI so we don't need this here. Also, this will cause issues for people using the module. Hardcoding the region means, the bucket will be deployed into that region, even if it's not what they want

Reply:
Copy that!...Done.